frontend/src/screens/settings/Backup.tsx

```diff
@@ -0,0 +1,342 @@
+            </form>
+          </div>
+          <MnemonicDialog
+            open={isDialogOpen}
+            onOpenChange={setIsDialogOpen}
+            mnemonic={decryptedMnemonic}
+          />
+        </div>
+      )}
+      <>
+        <Separator className="my-6" />
+        <div className="flex flex-col gap-8">
+          <div>
+            <h3 className="text-lg font-medium">Channels Backup</h3>
+            <p className="text-sm text-muted-foreground">
+              Your spending balance is stored in your lightning channels. In
+              case of recovery of your Alby Hub, they need to be backed up every
+              time you open a new channel.
+            </p>
+          </div>
+
+          <div>
+            <div className="flex gap-2 mb-1 items-center">
+              <h3 className="text-sm font-medium">Automated Channels Backup</h3>
+              {info?.albyAccountConnected ? (
+                <Badge variant={"positive"}>Active</Badge>
+              ) : (
+                <Badge>Recommended</Badge>
+              )}
+            </div>
+            {info?.albyAccountConnected ? (
+              <>
+                <p className="text-muted-foreground text-sm mb-8">
+                  Your channel state is backed up automatically after each
+                  channel creation. Using an external recovery tool and your
+                  recovery phrase, you can request channel recovery.
+                </p>
+                {info?.vssSupported && (
+                  <>
+                    <div>
+                      <div className="flex gap-2 mb-1 items-center">
+                        <h3 className="text-sm font-medium">
+                          Dynamic Channels Backup With Instant Recovery
+                        </h3>
+                        {me?.subscription.plan_code && info.ldkVssEnabled ? (
+                          <Badge variant={"positive"}>Active</Badge>
+                        ) : (
+                          <Badge>Alby Cloud</Badge>
+                        )}
+                      </div>
+                      <p className="text-sm text-muted-foreground mb-4">
+                        When enabled, your channels state is dynamically updated
+                        and stored end-to-end encrypted by Alby's Versioned
+                        Storage Service. This allows you to recover your
+                        spending balance with your recovery phrase alone,
+                        without having to close your channels.
+                      </p>
+
+                      {!info.ldkVssEnabled &&
+                        (!me?.subscription.plan_code ? (
+                          <ExternalLinkButton
+                            variant="secondary"
+                            size={"lg"}
+                            to="https://getalby.com/subscription/new"
+                          >
+                            Enable Dynamic Channels Backup
+                          </ExternalLinkButton>
```

Comment:
Maybe this can open this popup now (with some button text change)? cc @rolznz

<img width="400" alt="Screenshot 2025-03-20 at 5 57 37 PM" src="https://github.com/user-attachments/assets/4439781b-b693-42c2-948e-a94b0ea9fc42" />

Reply:
@im-adithya does the button text need to change? reusing this dialog would be great though :+1:

Reply:
"Upgrade to Enable Dynamic Channel Backup" will work for button

Reply:
@pavanjoshi914 we already show this upgrade dialog on click. So I don't think we need "Upgrade to" in the button text